src/server/bg_services/replication_server.js

```diff
@@ -100,7 +100,7 @@ async function copy_objects_mixed_types(req) {
                 try {
                     await noobaa_con.copyObject(params).promise();
                     copy_res.num_of_objects += 1;
-                    copy_res.size_of_objects += keys_diff_map[key][i].Size;
+                    copy_res.size_of_objects += keys_diff_map[key][i].ContentLength;
```

Comment:
This will not work, The only valid option here is when we have versions (key.length > 1) 
In "version" we do not have `ContentLength`, we have `Size`. 
https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectVersions.html